### PR TITLE
Roman numeral

### PR DESCRIPTION
**Pull request by sterlingking1990**
#This is my solution of the roman numeral task

### CODE REVIEW

File: src/transpose/index.js [hunk 1]
```diff
@@ -4,6 +4,34 @@
...
-function transpose(array) {}
+function transpose(array) {
+    //given an array A to be size nxm the transposed array B will 
+    //have number of row to be A[0].length and number of column to be
+    // A.length
+    //Having gotten row and column size of array B; 
+    //The Transposed array Bji=Aij
+
+    let rowSize=array[0].length
+    let matB=new Array(rowSize)
```
Comment (bondz) [inline src/transpose/index.js:15]: Prefer `Array.from` over `new Array`

**Comment by bondz** [comment]
35 commits though? Phew!

**Comment by sterlingking1990** [comment]
> Thank you very so much for your feedback, I will consider your
suggestions and learn more on how to improve.

**Comment by sterlingking1990** [comment]
I guess my commit is much? before now I hardly commit, but having learnt
about git and still learning more...

I would also love to know your approach on committing changes in chunks.

Thank you

**Pull request merged**